internal/manifest/manifest.go

```diff
@@ -86,13 +84,15 @@ type Kubernetes struct {
 }
 
 type DynamicNodePool struct {
-	Name             string           `validate:"required" yaml:"name"`
-	ProviderSpec     ProviderSpec     `validate:"required" yaml:"providerSpec"`
-	Count            int32            `validate:"required_without=AutoscalerConfig,excluded_with=AutoscalerConfig" yaml:"count"`
-	ServerType       string           `validate:"required" yaml:"serverType"`
-	Image            string           `validate:"required" yaml:"image"`
-	StorageDiskSize  int64            `validate:"omitempty,gte=50" yaml:"storageDiskSize"`
-	AutoscalerConfig AutoscalerConfig `validate:"required_without=Count,excluded_with=Count" yaml:"autoscaler"`
+	Name             string            `validate:"required" yaml:"name"`
+	ProviderSpec     ProviderSpec      `validate:"required" yaml:"providerSpec"`
+	Count            int32             `validate:"required_without=AutoscalerConfig,excluded_with=AutoscalerConfig" yaml:"count"`
+	ServerType       string            `validate:"required" yaml:"serverType"`
+	Image            string            `validate:"required" yaml:"image"`
+	StorageDiskSize  int64             `validate:"omitempty,gte=50" yaml:"storageDiskSize"`
+	AutoscalerConfig AutoscalerConfig  `validate:"required_without=Count,excluded_with=Count" yaml:"autoscaler"`
+	Labels           map[string]string `validate:"omitempty" yaml:"labels"`
```

Comment:
Validate labels with DNS compatible [naming convention?](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)